api/lib/domain/services/certification-candidates-ods-service.js

```diff
@@ -80,6 +80,7 @@ async function extractCertificationCandidatesFromCandidatesImportSheet({
           hasPixPlusDroit,
           hasPixPlusEdu1erDegre,
           hasPixPlusEdu2ndDegre,
+          hasPixPlusSante,
```

Comment:
Je propose de compléter le test `should return extracted and validated certification candidates with complementary certification` dans `api/tests/integration/domain/services/certification-candidates-ods-service/certification-candidates-ods-service_test.js`, car sinon on aura test qui couvre le cas ProSanté pour plus tard